newIDE/app/src/Profile/Subscription/PlanCard.js

```diff
@@ -159,19 +171,24 @@ const getPlanPrice = (
   }
 };
 
+const extrapolateMonthlyPricingSystemToYearlyBasis = (
+  monthlyPricingSystem: SubscriptionPlanPricingSystem
+): SubscriptionPlanPricingSystem => ({
+  ...monthlyPricingSystem,
+  amountInCents: monthlyPricingSystem.amountInCents * 12,
+  period: 'year',
+});
+
 export const getPlanPrices = ({
   pricingSystems,
-  hidePrice,
 }: {
   pricingSystems: SubscriptionPlanPricingSystem[],
-  hidePrice?: boolean,
 }): React.Node => {
-  if (hidePrice) return null;
```

Comment:
just checking, not relevant anymore?

Reply:
Oh yes! I didn't see it was exported and since the type was not strict, Flow didn't fail. Thanks!